images/linux/toolsets/toolset-1804.json

```diff
@@ -42,12 +42,11 @@
             "arch": "x64",
             "platform" : "linux",
             "versions": [
-                "1.15.*",
                 "1.16.*",
                 "1.17.*",
                 "1.18.*"
             ],
-            "default": "1.15.*"
+            "default": "1.17.*"
```

Comment:
Could you please confirm that we have an announcement for this update?
According to our docs we should publish announcement at least 2 weeks prior update default versions: https://github.com/actions/virtual-environments/blob/main/docs/software-and-images-guidelines.md#software-default-versions-update-policy-for-tools-with-multiple-versions-installed

Reply:
Here it is https://github.com/actions/virtual-environments/issues/5280

Reply:
But maybe we should update the title as we mention the default version change only in the description

Reply:
Ah, makes sense. 
Actually, only default version really matters for customers because all other versions are "tool-cache". It means they can be used only via setup-tasks. But if customers are using setup-tasks - they won't be affected at all because tasks will download version on-flight. So real impact is only default version change.